index.html

```diff
@@ -294,53 +294,89 @@ <h1 class="title">About
+                            <p class="lang-text">Frameworks</p>
+                            <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20"
+                                fill="currentColor">
+                                <path fill-rule="evenodd"
+                                    d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                                    clip-rule="evenodd" />
+                            </svg>
                         </div>
-                        <div class="lang">
-                            <img alt="css" src="/assets/css.png">
-                            <p>CSS</p>
+                    </li>
+                    <li class="list">
+                        <div class="dropdown-menu border-b">
+                            <p class="lang-text">Skills</p>
+                            <svg xmlns="http://www.w3.org/2000/svg" class="icon" viewBox="0 0 20 20"
+                                fill="currentColor">
+                                <path fill-rule="evenodd"
+                                    d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
+                                    clip-rule="evenodd" />
+                            </svg>
                         </div>
-                    </div>
-                </div>
-                <div class="language-section">
-                    <div class="dropdown-menu border-b">
-                        <p class="lang-text">Frameworks</p>
-                        <svg xmlns="http://www.w3.org/2000/svg" class="icon"
-                        viewBox="0 0 20 20" fill="currentColor">
-                        <path fill-rule="evenodd"
-                            d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-                            clip-rule="evenodd" />
-                    </svg>
-                    </div>
-                </div>
-                <div class="language-section">
-                    <div class="dropdown-menu border-b">
-                        <p class="lang-text">Skills</p>
-                        <svg xmlns="http://www.w3.org/2000/svg" class="icon"
-                        viewBox="0 0 20 20" fill="currentColor">
-                        <path fill-rule="evenodd"
-                            d="M7.293 14.707a1 1 0 010-1.414L10.586 10 7.293 6.707a1 1 0 011.414-1.414l4 4a1 1 0 010 1.414l-4 4a1 1 0 01-1.414 0z"
-                            clip-rule="evenodd" />
-                    </svg>
-                    </div>
-                </div>
+                    </li>
+                </ul>
             </div>
         </section>
+        <footer class="contact-section-bg">
+            <div class="contact-section">
+                <h1 class="title">Contact me</h1>
```

Comment:
- [ ] Please use `<h2>` tag instead of `<h1>` because you only allowed to use one `<h1>` tag per page for semantic and SEO reasons.

Reply:
thanks 😄